public/app/plugins/panel/geomap/layers/data/markersLayer.tsx

```diff
@@ -173,6 +177,55 @@ export const markersLayer: MapLayerRegistryItem<MarkersConfig> = {
               step: 0.1,
             },
           })
+          .addCustomEditor({
+            id: 'config.style.text',
+            path: 'config.style.text',
+            name: 'Text label',
+            editor: TextDimensionEditor,
+            defaultValue: defaultOptions.style.text,
+          })
+          .addNumberInput({
+            name: 'Text font size',
+            path: 'config.style.textConfig.fontSize',
+            defaultValue: defaultOptions.style.textConfig?.fontSize,
+          })
+          .addNumberInput({
+            name: 'Text X offset',
+            path: 'config.style.textConfig.offsetX',
+            defaultValue: 0,
+          })
+          .addNumberInput({
+            name: 'Text Y offset',
```

Comment:
When testing this locally it seems that positive numbers for y pushes text down instead of what a user may expect (up). Is there any way we could reverse how the y direction applies?

Reply:
Hmm I would guess that this depends on the user. 

This behavior falls in line with the canvas coordinate system and is also the default in [open layers](https://openlayers.org/en/latest/apidoc/module-ol_style_Text-Text.html)

We could multiply the y value by -1 but I would advocate that we don't change this default behavior.